apps/web/components/dashboard/preview/EditableTitle.tsx

```diff
@@ -35,8 +35,8 @@ export function EditableTitle({ bookmark }: { bookmark: ZBookmark }) {
   return (
     <EditableText
       originalText={title}
-      editClassName="p-2 text-center text-lg"
-      viewClassName="line-clamp-2 text-lg"
+      editClassName="p-2 text-center text-lg break-all"
+      viewClassName="inline line-clamp-2 text-lg text-ellipsis"
```

Comment:
I ended up going with a combo of `break-all` for the editable field and `break-words` for the view field, as mentioned above. It seems to work fine with all titles that I've tested on both desktop and mobile, but let me know if you notice any edge cases that don't work properly! (I also took out the text centering, since multi-line titles that truncate with an ellipsis look pretty strange to my eye… just my personal preference, but easy enough to put back if it's not to your liking!)